deep_speech_2/deploy/ctc_decoders.cpp

```diff
@@ -0,0 +1,337 @@
+#include <iostream>
+#include <map>
+#include <algorithm>
+#include <utility>
+#include <cmath>
+#include <limits>
+#include "fst/fstlib.h"
+#include "ctc_decoders.h"
+#include "decoder_utils.h"
+#include "path_trie.h"
+#include "ThreadPool.h"
+
+std::string ctc_best_path_decoder(std::vector<std::vector<double> > probs_seq,
```

Comment:
1. Let's use `ctc_greedy_decoder` instead.
2. Add a function doc
3. use `const std::vector<xxx>&` for arguments.

Reply:
1. Done
2. function doc was in the header
3. Done